diff --git a/services/labimagingdentaloptical/models/order_detail.js b/services/labimagingdentaloptical/models/order_detail.js
--- a/services/labimagingdentaloptical/models/order_detail.js
+++ b/services/labimagingdentaloptical/models/order_detail.js
@@ -92,1 +92,1 @@
-        order_schedule_confirm: {
+    order_schedule_confirm: {
diff --git a/services/labimagingdentaloptical/models/logs.js b/services/labimagingdentaloptical/models/logs.js
--- a/services/labimagingdentaloptical/models/logs.js
+++ b/services/labimagingdentaloptical/models/logs.js
@@ -21,1 +21,1 @@
-        userAddress : {
+    userAddress: {
diff --git a/services/superadmin/helpers/verifyToken.js b/services/superadmin/helpers/verifyToken.js
--- a/services/superadmin/helpers/verifyToken.js
+++ b/services/superadmin/helpers/verifyToken.js
@@ -30,1 +30,1 @@
-        //         message: messages.notAuthorized,
+    //         message: messages.notAuthorized,
@@ -41,1 +41,1 @@
-        req.user = decode.data
+    req.user = decode.data;
diff --git a/services/labimagingdentaloptical/validator/menu.js b/services/labimagingdentaloptical/validator/menu.js
--- a/services/labimagingdentaloptical/validator/menu.js
+++ b/services/labimagingdentaloptical/validator/menu.js
@@ -18,1 +18,1 @@
-const editPermValidator=[
+const editPermValidator = [
diff --git a/services/patient/models/subscription/purchasestatus.js b/services/patient/models/subscription/purchasestatus.js
--- a/services/patient/models/subscription/purchasestatus.js
+++ b/services/patient/models/subscription/purchasestatus.js
@@ -5,1 +5,1 @@
-        subscription_plan_name: {
+    subscription_plan_name: {
@@ -64,1 +64,1 @@
-            default:''
+      default: "",
@@ -67,1 +67,1 @@
-    { timestamps: true }
+  { timestamps: true }
diff --git a/services/superadmin/controllers/superadmin/staffManagementController.js b/services/superadmin/controllers/superadmin/staffManagementController.js
--- a/services/superadmin/controllers/superadmin/staffManagementController.js
+++ b/services/superadmin/controllers/superadmin/staffManagementController.js
@@ -27,1 +27,1 @@
-        Bucket: 'healthcare-crm-stage-docs',
+        Bucket: "healthcare-crm-stage-docs",
diff --git a/services/gateway/routes/paymentRoute.js b/services/gateway/routes/paymentRoute.js
--- a/services/gateway/routes/paymentRoute.js
+++ b/services/gateway/routes/paymentRoute.js
@@ -7,1 +7,1 @@
-paymentRoute.post("/create-payment-intent", createPaymentIntent)
+paymentRoute.post("/create-payment-intent", createPaymentIntent);
diff --git a/services/superadmin/controllers/menus/menu.js b/services/superadmin/controllers/menus/menu.js
--- a/services/superadmin/controllers/menus/menu.js
+++ b/services/superadmin/controllers/menus/menu.js
@@ -29,1 +29,1 @@
-            handleResponse(req, res, 200, {
+      handleResponse(req, res, 200, {
@@ -37,1 +37,1 @@
-    } catch (error) {
+  } catch (error) {
@@ -62,1 +62,1 @@
-            errorCode: "INTERNAL_SERVER_ERROR",
+      errorCode: "INTERNAL_SERVER_ERROR",
diff --git a/services/patient/models/profile_info.js b/services/patient/models/profile_info.js
--- a/services/patient/models/profile_info.js
+++ b/services/patient/models/profile_info.js
@@ -31,1 +31,1 @@
-            default:''
+      default: "",
diff --git a/services/insurance/models/medicineClaim/subscriberUseLimit.js b/services/insurance/models/medicineClaim/subscriberUseLimit.js
--- a/services/insurance/models/medicineClaim/subscriberUseLimit.js
+++ b/services/insurance/models/medicineClaim/subscriberUseLimit.js
@@ -30,1 +30,1 @@
-            type: Number,
+      type: Number,
diff --git a/services/pharmacy/controllers/Chat-Controller/Chat.js b/services/pharmacy/controllers/Chat-Controller/Chat.js
--- a/services/pharmacy/controllers/Chat-Controller/Chat.js
+++ b/services/pharmacy/controllers/Chat-Controller/Chat.js
@@ -16,1 +16,1 @@
-    }
+    };
@@ -28,1 +28,1 @@
-      isGroupChat:false
+      isGroupChat: false,
@@ -179,1 +179,1 @@
-        chat.profile_pic = GroupProfilePic;
+          chat.profile_pic = GroupProfilePic;
diff --git a/services/labimagingdentaloptical/middleware/sendSms.js b/services/labimagingdentaloptical/middleware/sendSms.js
--- a/services/labimagingdentaloptical/middleware/sendSms.js
+++ b/services/labimagingdentaloptical/middleware/sendSms.js
@@ -11,1 +11,1 @@
-            url: smsUrl,
+      url: smsUrl,
diff --git a/services/hospital/models/basic_info.js b/services/hospital/models/basic_info.js
--- a/services/hospital/models/basic_info.js
+++ b/services/hospital/models/basic_info.js
@@ -20,1 +20,1 @@
-        designation: {
+    designation: {
diff --git a/services/pharmacy/models/medicineClaim/medicineDetailsOnClaim.js b/services/pharmacy/models/medicineClaim/medicineDetailsOnClaim.js
--- a/services/pharmacy/models/medicineClaim/medicineDetailsOnClaim.js
+++ b/services/pharmacy/models/medicineClaim/medicineDetailsOnClaim.js
@@ -12,1 +12,1 @@
-        quantityPrescribed: {
+    quantityPrescribed: {
diff --git a/services/insurance/models/medicineClaim/commonInfo.js b/services/insurance/models/medicineClaim/commonInfo.js
--- a/services/insurance/models/medicineClaim/commonInfo.js
+++ b/services/insurance/models/medicineClaim/commonInfo.js
@@ -35,1 +35,1 @@
-            type: String
+      type: String,
@@ -60,1 +60,1 @@
-                "Paramedical-Professions-order",
+        "Paramedical-Professions-order",
diff --git a/services/superadmin/controllers/hospital/hopsital.validate.js b/services/superadmin/controllers/hospital/hopsital.validate.js
--- a/services/superadmin/controllers/hospital/hopsital.validate.js
+++ b/services/superadmin/controllers/hospital/hopsital.validate.js
@@ -11,1 +11,1 @@
-    check("action")
+  check("action")
diff --git a/services/gateway/controller/patientController.js b/services/gateway/controller/patientController.js
--- a/services/gateway/controller/patientController.js
+++ b/services/gateway/controller/patientController.js
@@ -35,1 +35,1 @@
-        headers: { role: req.header("role") }
+    headers: { role: req.header("role") },
@@ -132,1 +132,1 @@
-        url: `${baseurl}/patient/create-profile/add-vitals`,
+    url: `${baseurl}/patient/create-profile/add-vitals`,
diff --git a/services/labimagingdentaloptical/models/assessment.js b/services/labimagingdentaloptical/models/assessment.js
--- a/services/labimagingdentaloptical/models/assessment.js
+++ b/services/labimagingdentaloptical/models/assessment.js
@@ -25,1 +25,1 @@
-    { timestamps: true }
+  { timestamps: true }
diff --git a/services/superadmin/models/contentManagement/aboutUs.js b/services/superadmin/models/contentManagement/aboutUs.js
--- a/services/superadmin/models/contentManagement/aboutUs.js
+++ b/services/superadmin/models/contentManagement/aboutUs.js
@@ -4,1 +4,1 @@
-    {
+  {
diff --git a/services/patient/auth/auth.js b/services/patient/auth/auth.js
--- a/services/patient/auth/auth.js
+++ b/services/patient/auth/auth.js
@@ -37,1 +37,1 @@
-        if (!isMatch)
+    if (!isMatch)
diff --git a/services/superadmin/controllers/menus/userMenu_permission.js b/services/superadmin/controllers/menus/userMenu_permission.js
--- a/services/superadmin/controllers/menus/userMenu_permission.js
+++ b/services/superadmin/controllers/menus/userMenu_permission.js
@@ -38,1 +38,1 @@
-                status: true,
+        status: true,
diff --git a/services/pharmacy/models/onDutyGroup.js b/services/pharmacy/models/onDutyGroup.js
--- a/services/pharmacy/models/onDutyGroup.js
+++ b/services/pharmacy/models/onDutyGroup.js
@@ -5,1 +5,1 @@
-    {
+  {
diff --git a/services/patient/helpers/httpservice.js b/services/patient/helpers/httpservice.js
--- a/services/patient/helpers/httpservice.js
+++ b/services/patient/helpers/httpservice.js
@@ -46,1 +46,1 @@
-    post(path, data, headers, service){
+  post(path, data, headers, service) {
diff --git a/services/hospital/controllers/payment/payment.validate.js b/services/hospital/controllers/payment/payment.validate.js
--- a/services/hospital/controllers/payment/payment.validate.js
+++ b/services/hospital/controllers/payment/payment.validate.js
@@ -12,1 +12,1 @@
-    (req, res, next) => {
+  (req, res, next) => {
diff --git a/services/superadmin/models/superadmin/document_info.js b/services/superadmin/models/superadmin/document_info.js
--- a/services/superadmin/models/superadmin/document_info.js
+++ b/services/superadmin/models/superadmin/document_info.js
@@ -4,1 +4,1 @@
-        name: {
+    name: {
diff --git a/services/superadmin/controllers/subscription/subscriptionController.js b/services/superadmin/controllers/subscription/subscriptionController.js
--- a/services/superadmin/controllers/subscription/subscriptionController.js
+++ b/services/superadmin/controllers/subscription/subscriptionController.js
@@ -23,1 +23,1 @@
-            });
+      });
diff --git a/services/labimagingdentaloptical/controllers/staffmanagement.js b/services/labimagingdentaloptical/controllers/staffmanagement.js
--- a/services/labimagingdentaloptical/controllers/staffmanagement.js
+++ b/services/labimagingdentaloptical/controllers/staffmanagement.js
@@ -31,1 +31,1 @@
-            last_name,
+      last_name,
diff --git a/services/hospital/controllers/roles/role.js b/services/hospital/controllers/roles/role.js
--- a/services/hospital/controllers/roles/role.js
+++ b/services/hospital/controllers/roles/role.js
@@ -7,1 +7,1 @@
-let seqId=0;
+let seqId = 0;
@@ -48,1 +48,1 @@
-                        status: false,
+            status: false,
diff --git a/services/patient/middleware/verifyToken.js b/services/patient/middleware/verifyToken.js
--- a/services/patient/middleware/verifyToken.js
+++ b/services/patient/middleware/verifyToken.js
@@ -19,1 +19,1 @@
-        // if(decode.data.role !== role) return sendResponse(req, res, messageID.unAuthorizedUser, {
+    // if(decode.data.role !== role) return sendResponse(req, res, messageID.unAuthorizedUser, {
diff --git a/services/patient/app.js b/services/patient/app.js
--- a/services/patient/app.js
+++ b/services/patient/app.js
@@ -7,1 +7,1 @@
-const useragent = require('express-useragent');
+const useragent = require("express-useragent");
diff --git a/services/insurance/models/subscription/purchasestatus.js b/services/insurance/models/subscription/purchasestatus.js
--- a/services/insurance/models/subscription/purchasestatus.js
+++ b/services/insurance/models/subscription/purchasestatus.js
@@ -7,1 +7,1 @@
-        },
+    },
diff --git a/services/labimagingdentaloptical/controllers/eprescription.js b/services/labimagingdentaloptical/controllers/eprescription.js
--- a/services/labimagingdentaloptical/controllers/eprescription.js
+++ b/services/labimagingdentaloptical/controllers/eprescription.js
@@ -6,1 +6,1 @@
-const httpService = new Http()
+const httpService = new Http();
diff --git a/services/labimagingdentaloptical/controllers/templatebuilder.js b/services/labimagingdentaloptical/controllers/templatebuilder.js
--- a/services/labimagingdentaloptical/controllers/templatebuilder.js
+++ b/services/labimagingdentaloptical/controllers/templatebuilder.js
@@ -117,1 +117,1 @@
-          var keynew = sort.split(":")[0];
+        var keynew = sort.split(":")[0];
diff --git a/services/patient/config/constants.js b/services/patient/config/constants.js
--- a/services/patient/config/constants.js
+++ b/services/patient/config/constants.js
@@ -27,1 +27,1 @@
-    EMAIL: {
+  EMAIL: {
@@ -50,1 +50,1 @@
-    loginSuccess: "Login Successfully",
+  loginSuccess: "Login Successfully",
diff --git a/services/hospital/routes/leave.js b/services/hospital/routes/leave.js
--- a/services/hospital/routes/leave.js
+++ b/services/hospital/routes/leave.js
@@ -13,1 +13,1 @@
-leaveManagementRoute.get("/doctorstaffleave-list",getAllMyDoctorStaffLeave);
+leaveManagementRoute.get("/doctorstaffleave-list", getAllMyDoctorStaffLeave);
diff --git a/services/insurance/controllers/hospital/hospital.validate.js b/services/insurance/controllers/hospital/hospital.validate.js
--- a/services/insurance/controllers/hospital/hospital.validate.js
+++ b/services/insurance/controllers/hospital/hospital.validate.js
@@ -7,1 +7,1 @@
-        .withMessage("Parent ID missing")
+    .withMessage("Parent ID missing")
diff --git a/services/insurance/controllers/roles/role.js b/services/insurance/controllers/roles/role.js
--- a/services/insurance/controllers/roles/role.js
+++ b/services/insurance/controllers/roles/role.js
@@ -48,1 +48,1 @@
-                        status: false,
+            status: false,
diff --git a/services/labimagingdentaloptical/models/eprescription_other.js b/services/labimagingdentaloptical/models/eprescription_other.js
--- a/services/labimagingdentaloptical/models/eprescription_other.js
+++ b/services/labimagingdentaloptical/models/eprescription_other.js
@@ -21,1 +21,1 @@
-        reason_for_other: {
+    reason_for_other: {
diff --git a/services/insurance/helpers/httpservice.js b/services/insurance/helpers/httpservice.js
--- a/services/insurance/helpers/httpservice.js
+++ b/services/insurance/helpers/httpservice.js
@@ -25,1 +25,1 @@
-        })
+    });
diff --git a/services/labimagingdentaloptical/controllers/Chat.js b/services/labimagingdentaloptical/controllers/Chat.js
--- a/services/labimagingdentaloptical/controllers/Chat.js
+++ b/services/labimagingdentaloptical/controllers/Chat.js
@@ -25,1 +25,1 @@
-            getData = await getIndividualFourPortalStaff(loggedInId, adminId);
+      getData = await getIndividualFourPortalStaff(loggedInId, adminId);
@@ -49,1 +49,1 @@
-            errorCode: "INTERNAL_SERVER_ERROR",
+      errorCode: "INTERNAL_SERVER_ERROR",
diff --git a/services/hospital/middleware/utils.js b/services/hospital/middleware/utils.js
--- a/services/hospital/middleware/utils.js
+++ b/services/hospital/middleware/utils.js
@@ -29,1 +29,1 @@
-    return encryptedData;
+  return encryptedData;
diff --git a/services/insurance/config/constants.js b/services/insurance/config/constants.js
--- a/services/insurance/config/constants.js
+++ b/services/insurance/config/constants.js
@@ -5,1 +5,1 @@
-        HOST: process.env.MONGO_DB_HOST ,
+    HOST: process.env.MONGO_DB_HOST,
@@ -37,1 +37,1 @@
-    SMS_API_KEY: process.env.SMS_API_KEY ||"",
+  SMS_API_KEY: process.env.SMS_API_KEY || "",
diff --git a/services/insurance/middleware/sendSms.js b/services/insurance/middleware/sendSms.js
--- a/services/insurance/middleware/sendSms.js
+++ b/services/insurance/middleware/sendSms.js
@@ -10,1 +10,1 @@
-            method: "get",
+      method: "get",
diff --git a/services/patient/models/immunization_info.js b/services/patient/models/immunization_info.js
--- a/services/patient/models/immunization_info.js
+++ b/services/patient/models/immunization_info.js
@@ -23,1 +23,1 @@
-        route_of_administered: {
+    route_of_administered: {
diff --git a/services/labimagingdentaloptical/constant.js b/services/labimagingdentaloptical/constant.js
--- a/services/labimagingdentaloptical/constant.js
+++ b/services/labimagingdentaloptical/constant.js
@@ -13,1 +13,1 @@
-    userExist: "User already exist.",
+  userExist: "User already exist.",
diff --git a/services/pharmacy/models/subscription/purchasestatus.js b/services/pharmacy/models/subscription/purchasestatus.js
--- a/services/pharmacy/models/subscription/purchasestatus.js
+++ b/services/pharmacy/models/subscription/purchasestatus.js
@@ -15,1 +15,1 @@
-            type: Array,
+      type: Array,
diff --git a/services/pharmacy/models/location_info.js b/services/pharmacy/models/location_info.js
--- a/services/pharmacy/models/location_info.js
+++ b/services/pharmacy/models/location_info.js
@@ -20,1 +20,1 @@
-            type: mongoose.Schema.Types.ObjectId,
+      type: mongoose.Schema.Types.ObjectId,
@@ -79,1 +79,1 @@
-            ref: "PortalUser",
+      ref: "PortalUser",
diff --git a/services/superadmin/models/contentManagement/contactUs.js b/services/superadmin/models/contentManagement/contactUs.js
--- a/services/superadmin/models/contentManagement/contactUs.js
+++ b/services/superadmin/models/contentManagement/contactUs.js
@@ -7,1 +7,1 @@
-        // },
+    // },
@@ -19,1 +19,1 @@
-        }
+    },
diff --git a/services/hospital/models/educational_details.js b/services/hospital/models/educational_details.js
--- a/services/hospital/models/educational_details.js
+++ b/services/hospital/models/educational_details.js
@@ -30,1 +30,1 @@
-      ref: "PortalUser"
+      ref: "PortalUser",
diff --git a/services/labimagingdentaloptical/helpers/notification.js b/services/labimagingdentaloptical/helpers/notification.js
--- a/services/labimagingdentaloptical/helpers/notification.js
+++ b/services/labimagingdentaloptical/helpers/notification.js
@@ -25,1 +25,1 @@
-            axios({
+      axios({
diff --git a/services/hospital/helpers/ses.js b/services/hospital/helpers/ses.js
--- a/services/hospital/helpers/ses.js
+++ b/services/hospital/helpers/ses.js
@@ -2,1 +2,1 @@
-const AWS = require('aws-sdk');
+const AWS = require("aws-sdk");
@@ -20,1 +20,1 @@
-      Source: '<email address you verified>',
+    Source: "<email address you verified>",
@@ -22,1 +22,1 @@
-      Destination: {
+    Destination: {
diff --git a/services/pharmacy/middleware/verifyToken.js b/services/pharmacy/middleware/verifyToken.js
--- a/services/pharmacy/middleware/verifyToken.js
+++ b/services/pharmacy/middleware/verifyToken.js
@@ -27,1 +27,1 @@
-        //     return sendResponse(req, res, messageID.unAuthorizedUser, {
+    //     return sendResponse(req, res, messageID.unAuthorizedUser, {
diff --git a/services/gateway/routes/superAdminRoute.js b/services/gateway/routes/superAdminRoute.js
--- a/services/gateway/routes/superAdminRoute.js
+++ b/services/gateway/routes/superAdminRoute.js
@@ -17,1 +17,1 @@
-    getInsuranceAdminDetails,
+  getInsuranceAdminDetails,
diff --git a/services/hospital/models/portal_user.js b/services/hospital/models/portal_user.js
--- a/services/hospital/models/portal_user.js
+++ b/services/hospital/models/portal_user.js
@@ -13,1 +13,1 @@
-        },
+    },
diff --git a/services/gateway/controller/insurance/subscriberController.js b/services/gateway/controller/insurance/subscriberController.js
--- a/services/gateway/controller/insurance/subscriberController.js
+++ b/services/gateway/controller/insurance/subscriberController.js
@@ -20,1 +20,1 @@
-        form.append(key, req.body[key]);
+    form.append(key, req.body[key]);
diff --git a/services/hospital/models/assessment.js b/services/hospital/models/assessment.js
--- a/services/hospital/models/assessment.js
+++ b/services/hospital/models/assessment.js
@@ -15,1 +15,1 @@
-                    default:null
+          default: null,
@@ -21,1 +21,1 @@
-            require: true,
+      require: true,
diff --git a/services/patient/models/insurance_info.js b/services/patient/models/insurance_info.js
--- a/services/patient/models/insurance_info.js
+++ b/services/patient/models/insurance_info.js
@@ -42,1 +42,1 @@
-        //     reimbursement_rate: {
+    //     reimbursement_rate: {
@@ -100,1 +100,1 @@
-            type: mongoose.Schema.Types.ObjectId,
+      type: mongoose.Schema.Types.ObjectId,
diff --git a/services/hospital/controllers/individual_doctor_staff_controller.js b/services/hospital/controllers/individual_doctor_staff_controller.js
--- a/services/hospital/controllers/individual_doctor_staff_controller.js
+++ b/services/hospital/controllers/individual_doctor_staff_controller.js
@@ -11,1 +11,1 @@
-import crypto from "crypto"
+import crypto from "crypto";
diff --git a/services/insurance/controllers/insurance/card_Preview_master/card.js b/services/insurance/controllers/insurance/card_Preview_master/card.js
--- a/services/insurance/controllers/insurance/card_Preview_master/card.js
+++ b/services/insurance/controllers/insurance/card_Preview_master/card.js
@@ -72,1 +72,1 @@
-                        "_id.has_category": 1, 
+            "_id.has_category": 1,
@@ -98,1 +98,1 @@
-        } catch (error) {
+    } catch (error) {
diff --git a/services/hospital/helpers/transmission.js b/services/hospital/helpers/transmission.js
--- a/services/hospital/helpers/transmission.js
+++ b/services/hospital/helpers/transmission.js
@@ -18,1 +18,1 @@
-    // const ip = getIpFromRequest(req);
+  // const ip = getIpFromRequest(req);
diff --git a/services/superadmin/controllers/complaintManagement/complaintManagement.js b/services/superadmin/controllers/complaintManagement/complaintManagement.js
--- a/services/superadmin/controllers/complaintManagement/complaintManagement.js
+++ b/services/superadmin/controllers/complaintManagement/complaintManagement.js
@@ -71,1 +71,1 @@
-            var sort = req.query.sort
+      var sort = req.query.sort;
diff --git a/services/superadmin/controllers/speciality/speciality.validate.js b/services/superadmin/controllers/speciality/speciality.validate.js
--- a/services/superadmin/controllers/speciality/speciality.validate.js
+++ b/services/superadmin/controllers/speciality/speciality.validate.js
@@ -13,1 +13,1 @@
-        .withMessage("category missing")
+    .withMessage("category missing")
diff --git a/services/pharmacy/models/medicineClaim/commonInfo.js b/services/pharmacy/models/medicineClaim/commonInfo.js
--- a/services/pharmacy/models/medicineClaim/commonInfo.js
+++ b/services/pharmacy/models/medicineClaim/commonInfo.js
@@ -7,1 +7,1 @@
-        },
+    },
@@ -58,1 +58,1 @@
-        service: {
+    service: {
diff --git a/services/insurance/controllers/insurance/subscriber/subscriber.validate.js b/services/insurance/controllers/insurance/subscriber/subscriber.validate.js
--- a/services/insurance/controllers/insurance/subscriber/subscriber.validate.js
+++ b/services/insurance/controllers/insurance/subscriber/subscriber.validate.js
@@ -64,1 +64,1 @@
-      .withMessage("Card ID empty"),
+    .withMessage("Card ID empty"),
@@ -65,1 +65,1 @@
-    check("employee_id")
+  check("employee_id")
diff --git a/services/patient/controllers/payment/paymentController.js b/services/patient/controllers/payment/paymentController.js
--- a/services/patient/controllers/payment/paymentController.js
+++ b/services/patient/controllers/payment/paymentController.js
@@ -28,1 +28,1 @@
-        case 'half-yearly':
+    case "half-yearly":
diff --git a/services/labimagingdentaloptical/models/leave_management.js b/services/labimagingdentaloptical/models/leave_management.js
--- a/services/labimagingdentaloptical/models/leave_management.js
+++ b/services/labimagingdentaloptical/models/leave_management.js
@@ -48,1 +48,1 @@
-    }  // 0=pending , 1= approved ,2= rejected
+    }, // 0=pending , 1= approved ,2= rejected
diff --git a/services/pharmacy/helpers/transmission.js b/services/pharmacy/helpers/transmission.js
--- a/services/pharmacy/helpers/transmission.js
+++ b/services/pharmacy/helpers/transmission.js
@@ -81,1 +81,1 @@
-          errorCode: "INTERNAL_SERVER_ERROR",
+      errorCode: "INTERNAL_SERVER_ERROR",
diff --git a/services/labimagingdentaloptical/models/eprescription.js b/services/labimagingdentaloptical/models/eprescription.js
--- a/services/labimagingdentaloptical/models/eprescription.js
+++ b/services/labimagingdentaloptical/models/eprescription.js
@@ -49,1 +49,1 @@
-            default: false
+      default: false,
diff --git a/services/labimagingdentaloptical/helpers/transmission.js b/services/labimagingdentaloptical/helpers/transmission.js
--- a/services/labimagingdentaloptical/helpers/transmission.js
+++ b/services/labimagingdentaloptical/helpers/transmission.js
@@ -12,1 +12,1 @@
-        ""
+    ""
diff --git a/services/labimagingdentaloptical/controllers/email_invitation.js b/services/labimagingdentaloptical/controllers/email_invitation.js
--- a/services/labimagingdentaloptical/controllers/email_invitation.js
+++ b/services/labimagingdentaloptical/controllers/email_invitation.js
@@ -39,1 +39,1 @@
-                    }
+          },
diff --git a/services/patient/models/notification.js b/services/patient/models/notification.js
--- a/services/patient/models/notification.js
+++ b/services/patient/models/notification.js
@@ -4,1 +4,1 @@
-    {
+  {
diff --git a/services/pharmacy/models/admin_info.js b/services/pharmacy/models/admin_info.js
--- a/services/pharmacy/models/admin_info.js
+++ b/services/pharmacy/models/admin_info.js
@@ -11,1 +11,1 @@
-        middle_name: {
+    middle_name: {
@@ -60,1 +60,1 @@
-                    type: mongoose.Schema.Types.ObjectId,
+          type: mongoose.Schema.Types.ObjectId,
diff --git a/services/pharmacy/models/email_invitation.js b/services/pharmacy/models/email_invitation.js
--- a/services/pharmacy/models/email_invitation.js
+++ b/services/pharmacy/models/email_invitation.js
@@ -18,1 +18,1 @@
-        phone: {
+    phone: {
diff --git a/services/pharmacy/controllers/roles/role.js b/services/pharmacy/controllers/roles/role.js
--- a/services/pharmacy/controllers/roles/role.js
+++ b/services/pharmacy/controllers/roles/role.js
@@ -18,1 +18,1 @@
-                const newval = new roleCounter({ id: "autoval", seq: 1 })
+        const newval = new roleCounter({ id: "autoval", seq: 1 });
@@ -39,1 +39,1 @@
-                if (foundItems.length == 0) {
+        if (foundItems.length == 0) {
@@ -59,1 +59,1 @@
-                  message: "failed to add Role",
+          message: "failed to add Role",
diff --git a/services/pharmacy/models/reset_password_history.js b/services/pharmacy/models/reset_password_history.js
--- a/services/pharmacy/models/reset_password_history.js
+++ b/services/pharmacy/models/reset_password_history.js
@@ -11,1 +11,1 @@
-      required: true
+      required: true,
diff --git a/services/superadmin/controllers/roles/role.js b/services/superadmin/controllers/roles/role.js
--- a/services/superadmin/controllers/roles/role.js
+++ b/services/superadmin/controllers/roles/role.js
@@ -28,1 +28,1 @@
-    // } catch (error) {
+  // } catch (error) {
@@ -33,1 +33,1 @@
-    //         errorCode: "INTERNAL_SERVER_ERROR",
+  //         errorCode: "INTERNAL_SERVER_ERROR",
diff --git a/services/pharmacy/controllers/menus/userMenu_permission.js b/services/pharmacy/controllers/menus/userMenu_permission.js
--- a/services/pharmacy/controllers/menus/userMenu_permission.js
+++ b/services/pharmacy/controllers/menus/userMenu_permission.js
@@ -26,1 +26,1 @@
-            handleResponse(req, res, 200, {
+      handleResponse(req, res, 200, {
diff --git a/services/insurance/models/insurance/subscriber/subscriberPlanHistory.js b/services/insurance/models/insurance/subscriber/subscriberPlanHistory.js
--- a/services/insurance/models/insurance/subscriber/subscriberPlanHistory.js
+++ b/services/insurance/models/insurance/subscriber/subscriberPlanHistory.js
@@ -7,1 +7,1 @@
-            ref: "Plan",
+      ref: "Plan",
diff --git a/services/superadmin/controllers/superadmin/superadminController.js b/services/superadmin/controllers/superadmin/superadminController.js
--- a/services/superadmin/controllers/superadmin/superadminController.js
+++ b/services/superadmin/controllers/superadmin/superadminController.js
@@ -8,1 +8,1 @@
-const csv = require('fast-csv');
+const csv = require("fast-csv");
diff --git a/services/superadmin/controllers/common_data/commonDataController.js b/services/superadmin/controllers/common_data/commonDataController.js
--- a/services/superadmin/controllers/common_data/commonDataController.js
+++ b/services/superadmin/controllers/common_data/commonDataController.js
@@ -491,1 +491,1 @@
-          sortingarray[keynew] = value;
+        sortingarray[keynew] = value;
diff --git a/services/insurance/models/users/menu_model.js b/services/insurance/models/users/menu_model.js
--- a/services/insurance/models/users/menu_model.js
+++ b/services/insurance/models/users/menu_model.js
@@ -8,1 +8,1 @@
-    parent_id:{
+    parent_id: {
diff --git a/services/patient/helpers/cloudfront.js b/services/patient/helpers/cloudfront.js
--- a/services/patient/helpers/cloudfront.js
+++ b/services/patient/helpers/cloudfront.js
@@ -14,1 +14,1 @@
-    correctClockSkew: true
+  correctClockSkew: true,
diff --git a/services/superadmin/models/superadmin/portal_user.js b/services/superadmin/models/superadmin/portal_user.js
--- a/services/superadmin/models/superadmin/portal_user.js
+++ b/services/superadmin/models/superadmin/portal_user.js
@@ -44,1 +44,1 @@
-        dob: {
+    dob: {
diff --git a/services/superadmin/models/superadmin/Chat/Message.js b/services/superadmin/models/superadmin/Chat/Message.js
--- a/services/superadmin/models/superadmin/Chat/Message.js
+++ b/services/superadmin/models/superadmin/Chat/Message.js
@@ -24,1 +24,1 @@
-      default: false
+      default: false,
diff --git a/services/hospital/models/fee_management.js b/services/hospital/models/fee_management.js
--- a/services/hospital/models/fee_management.js
+++ b/services/hospital/models/fee_management.js
@@ -30,1 +30,1 @@
-        }
+        },
diff --git a/services/superadmin/helpers/firebase_notification.js b/services/superadmin/helpers/firebase_notification.js
--- a/services/superadmin/helpers/firebase_notification.js
+++ b/services/superadmin/helpers/firebase_notification.js
@@ -8,1 +8,1 @@
-        var message = {
+    var message = {
diff --git a/services/labimagingdentaloptical/models/reminder.js b/services/labimagingdentaloptical/models/reminder.js
--- a/services/labimagingdentaloptical/models/reminder.js
+++ b/services/labimagingdentaloptical/models/reminder.js
@@ -6,1 +6,1 @@
-            type: String,
+      type: String,
diff --git a/services/insurance/models/insurance/subscriber/subscriber.js b/services/insurance/models/insurance/subscriber/subscriber.js
--- a/services/insurance/models/insurance/subscriber/subscriber.js
+++ b/services/insurance/models/insurance/subscriber/subscriber.js
@@ -31,1 +31,1 @@
-        company_name: {
+    company_name: {
diff --git a/services/pharmacy/middleware/utils.js b/services/pharmacy/middleware/utils.js
--- a/services/pharmacy/middleware/utils.js
+++ b/services/pharmacy/middleware/utils.js
@@ -19,1 +19,1 @@
-    } else {
+  } else {
diff --git a/services/superadmin/models/contentManagement/privacyAndCondition.js b/services/superadmin/models/contentManagement/privacyAndCondition.js
--- a/services/superadmin/models/contentManagement/privacyAndCondition.js
+++ b/services/superadmin/models/contentManagement/privacyAndCondition.js
@@ -15,1 +15,1 @@
-    { timestamps: true }
+  { timestamps: true }
diff --git a/services/superadmin/middleware/httpservice.js b/services/superadmin/middleware/httpservice.js
--- a/services/superadmin/middleware/httpservice.js
+++ b/services/superadmin/middleware/httpservice.js
@@ -11,1 +11,1 @@
-        const baseurl = BaseUrl.userManagementServiceUrl;
+    const baseurl = BaseUrl.userManagementServiceUrl;
@@ -16,1 +16,1 @@
-                params: data,
+        params: data,
diff --git a/services/gateway/routes/proxy-route.js b/services/gateway/routes/proxy-route.js
--- a/services/gateway/routes/proxy-route.js
+++ b/services/gateway/routes/proxy-route.js
@@ -68,1 +68,1 @@
-        url: "/healthcare-crm-labimagingdentaloptical",
+    url: "/healthcare-crm-labimagingdentaloptical",
diff --git a/services/gateway/routes/hospital_route.js b/services/gateway/routes/hospital_route.js
--- a/services/gateway/routes/hospital_route.js
+++ b/services/gateway/routes/hospital_route.js
@@ -3,1 +3,1 @@
-const hospitalRoute = express.Router()
+const hospitalRoute = express.Router();
diff --git a/services/pharmacy/helpers/verifyToken.js b/services/pharmacy/helpers/verifyToken.js
--- a/services/pharmacy/helpers/verifyToken.js
+++ b/services/pharmacy/helpers/verifyToken.js
@@ -30,1 +30,1 @@
-        //         message: messages.notAuthorized,
+    //         message: messages.notAuthorized,
@@ -41,1 +41,1 @@
-        req.user = decode.data
+    req.user = decode.data;
diff --git a/services/patient/validator/menu.js b/services/patient/validator/menu.js
--- a/services/patient/validator/menu.js
+++ b/services/patient/validator/menu.js
@@ -19,1 +19,1 @@
-    ...permValidator,
+  ...permValidator,
diff --git a/services/insurance/middleware/constant.js b/services/insurance/middleware/constant.js
--- a/services/insurance/middleware/constant.js
+++ b/services/insurance/middleware/constant.js
@@ -104,1 +104,1 @@
-  `
+  `;
@@ -243,1 +243,1 @@
-      name: "New patient sign-ups"
+      name: "New patient sign-ups",
diff --git a/services/superadmin/models/superadmin/manual_medicine_claim.js b/services/superadmin/models/superadmin/manual_medicine_claim.js
--- a/services/superadmin/models/superadmin/manual_medicine_claim.js
+++ b/services/superadmin/models/superadmin/manual_medicine_claim.js
@@ -65,1 +65,1 @@
-        comments: {
+    comments: {
diff --git a/services/labimagingdentaloptical/controllers/roles/role.js b/services/labimagingdentaloptical/controllers/roles/role.js
--- a/services/labimagingdentaloptical/controllers/roles/role.js
+++ b/services/labimagingdentaloptical/controllers/roles/role.js
@@ -26,1 +26,1 @@
-                const list = rolesArray.map((singleData) => ({
+        const list = rolesArray.map((singleData) => ({
diff --git a/services/labimagingdentaloptical/models/appointment.js b/services/labimagingdentaloptical/models/appointment.js
--- a/services/labimagingdentaloptical/models/appointment.js
+++ b/services/labimagingdentaloptical/models/appointment.js
@@ -8,1 +8,1 @@
-        },
+    },
@@ -61,1 +61,1 @@
-        templateJSON: {
+    templateJSON: {
@@ -109,1 +109,1 @@
-            patientMobile: {
+      patientMobile: {
diff --git a/services/labimagingdentaloptical/models/location_info.js b/services/labimagingdentaloptical/models/location_info.js
--- a/services/labimagingdentaloptical/models/location_info.js
+++ b/services/labimagingdentaloptical/models/location_info.js
@@ -6,1 +6,1 @@
-            default: "Point"
+    default: "Point",
@@ -70,1 +70,1 @@
-            required: true,
+      required: true,
diff --git a/services/superadmin/models/common_data/village.js b/services/superadmin/models/common_data/village.js
--- a/services/superadmin/models/common_data/village.js
+++ b/services/superadmin/models/common_data/village.js
@@ -7,1 +7,1 @@
-        },
+    },
diff --git a/services/gateway/routes/userRoute.js b/services/gateway/routes/userRoute.js
--- a/services/gateway/routes/userRoute.js
+++ b/services/gateway/routes/userRoute.js
@@ -6,1 +6,1 @@
-import { getAllHealthPlan } from "../controller/insurance/healthPlanController"
+import { getAllHealthPlan } from "../controller/insurance/healthPlanController";
diff --git a/services/superadmin/validator/menu.js b/services/superadmin/validator/menu.js
--- a/services/superadmin/validator/menu.js
+++ b/services/superadmin/validator/menu.js
@@ -14,1 +14,1 @@
-    body("menu_id", "Please Enter a Valid MenuId").not().isEmpty(),
+  body("menu_id", "Please Enter a Valid MenuId").not().isEmpty(),
diff --git a/services/labimagingdentaloptical/models/order/order_detail.js b/services/labimagingdentaloptical/models/order/order_detail.js
--- a/services/labimagingdentaloptical/models/order/order_detail.js
+++ b/services/labimagingdentaloptical/models/order/order_detail.js
@@ -104,1 +104,1 @@
-                "Dental",
+        "Dental",
diff --git a/services/hospital/models/pathologyTestInfoNew.js b/services/hospital/models/pathologyTestInfoNew.js
--- a/services/hospital/models/pathologyTestInfoNew.js
+++ b/services/hospital/models/pathologyTestInfoNew.js
@@ -8,1 +8,1 @@
-        },
+    },
diff --git a/services/gateway/controller/readFile.js b/services/gateway/controller/readFile.js
--- a/services/gateway/controller/readFile.js
+++ b/services/gateway/controller/readFile.js
@@ -33,1 +33,1 @@
-                        res[key] = res[key].replace(/ +/g, "")
+            res[key] = res[key].replace(/ +/g, "");
@@ -35,1 +35,1 @@
-                    if (key == "service_limit") {
+          if (key == "service_limit") {
@@ -38,1 +38,1 @@
-                    if (key == "repayment_condition_max") {
+          if (key == "repayment_condition_max") {
@@ -41,1 +41,1 @@
-                    if (key == "repayment_condition_for") {
+          if (key == "repayment_condition_for") {
@@ -56,1 +56,1 @@
-    if (data.length == 0) {
+  if (data.length == 0) {
diff --git a/services/hospital/controllers/hospital_staff_controller.js b/services/hospital/controllers/hospital_staff_controller.js
--- a/services/hospital/controllers/hospital_staff_controller.js
+++ b/services/hospital/controllers/hospital_staff_controller.js
@@ -27,1 +27,1 @@
-    async addStaff(req, res) {
+  async addStaff(req, res) {
diff --git a/services/pharmacy/models/order/medicine_detail.js b/services/pharmacy/models/order/medicine_detail.js
--- a/services/pharmacy/models/order/medicine_detail.js
+++ b/services/pharmacy/models/order/medicine_detail.js
@@ -39,1 +39,1 @@
-        price_per_unit: {
+    price_per_unit: {
@@ -67,1 +67,1 @@
-    { timestamps: true }
+  { timestamps: true }
diff --git a/services/labimagingdentaloptical/models/questionnaire.js b/services/labimagingdentaloptical/models/questionnaire.js
--- a/services/labimagingdentaloptical/models/questionnaire.js
+++ b/services/labimagingdentaloptical/models/questionnaire.js
@@ -6,1 +6,1 @@
-            type: String,
+      type: String,
diff --git a/services/patient/models/immunization_list.js b/services/patient/models/immunization_list.js
--- a/services/patient/models/immunization_list.js
+++ b/services/patient/models/immunization_list.js
@@ -15,1 +15,1 @@
-            ref: "Superadmin"
+      ref: "Superadmin",
diff --git a/services/pharmacy/routes/pharmacy.js b/services/pharmacy/routes/pharmacy.js
--- a/services/pharmacy/routes/pharmacy.js
+++ b/services/pharmacy/routes/pharmacy.js
@@ -37,1 +37,1 @@
-    //         errorCode: "INTERNAL_SERVER_ERROR",
+  //         errorCode: "INTERNAL_SERVER_ERROR",
diff --git a/services/insurance/controllers/payment/payment.validate.js b/services/insurance/controllers/payment/payment.validate.js
--- a/services/insurance/controllers/payment/payment.validate.js
+++ b/services/insurance/controllers/payment/payment.validate.js
@@ -22,1 +22,1 @@
-        .withMessage("Please provide payment method type"),
+    .withMessage("Please provide payment method type"),
@@ -28,1 +28,1 @@
-        .withMessage("Please provide description"),
+    .withMessage("Please provide description"),
diff --git a/services/hospital/helpers/notification.js b/services/hospital/helpers/notification.js
--- a/services/hospital/helpers/notification.js
+++ b/services/hospital/helpers/notification.js
@@ -34,1 +34,1 @@
-                    const decryData = decryptionData(response.data)
+            const decryData = decryptionData(response.data);
diff --git a/services/hospital/controllers/masterController.js b/services/hospital/controllers/masterController.js
--- a/services/hospital/controllers/masterController.js
+++ b/services/hospital/controllers/masterController.js
@@ -62,1 +62,1 @@
-        console.log(req.body,"check body ------9");
+    console.log(req.body, "check body ------9");