diff --git a/services/insurance/controllers/hospital/user.js b/services/insurance/controllers/hospital/user.js
--- a/services/insurance/controllers/hospital/user.js
+++ b/services/insurance/controllers/hospital/user.js
@@ -67,1 +67,1 @@
-  };
+  }
diff --git a/services/hospital/helpers/ses.js b/services/hospital/helpers/ses.js
--- a/services/hospital/helpers/ses.js
+++ b/services/hospital/helpers/ses.js
@@ -27,1 +27,1 @@
-      TemplateData: '{ \"name\':\'John Doe\'}'
+      TemplateData: '{ "name\':\'John Doe\'}'